firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -184,11 +185,16 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         // Tap on each option in that menu
         mozWaitForElementToExist(app.buttons["Shopping.ReviewQualityCard.ExpandButton"])
         app.buttons["Shopping.ReviewQualityCard.ExpandButton"].tap()
+        app.swipeUp(velocity: 15)
         snapshot("ReviewQualityCard")
         app.buttons["Shopping.ReviewQualityCard.ExpandButton"].tap()
 
         mozWaitForElementToExist(app.buttons["Shopping.SettingsCard.ExpandButton"])
         app.buttons["Shopping.SettingsCard.ExpandButton"].tap()
         snapshot("SettingsCard")
+
+        app.buttons["Shopping.Sheet.CloseButton"].tap()
```

Comment:
I would not add this new step as it may add more complexity and flakiness to the tests while not showing new strings / screens... they are the same than in the full view